src/model/manifest.go

```diff
@@ -13,6 +14,51 @@ const (
 	SortByPath = "path"
 )
 
+type AllowedBranch string
+type AllowedBranches []AllowedBranch
+
+func (v AllowedBranches) String() string {
+	if len(v) == 0 {
+		return `[]`
+	}
+
+	items := make([]string, 0)
+	for _, item := range v {
+		items = append(items, string(item))
+	}
+	return `"` + strings.Join(items, `", "`) + `"`
+}
+
+func (v AllowedBranches) IsBranchAllowed(id int, name string) bool {
```

Comment:
trochu mi tu evokuje ze medzi parametrami je AND, tj musi platit aj id aj name. Cakal by som ze na vstupe bude `*model.Branch`, ale mozno sa to potom horsie testuje.

Reply:
mas pravdu, dam tam `model.Branch` v dalsom PR, ktory na to nadvazuje